### PR TITLE
refactor docs

### PR DESCRIPTION
**Pull request by tomer953**

### CODE REVIEW

File: docs/src/content/docs/utilities/navigation-end.md [hunk 1]
```diff
@@ -24,12 +26,34 @@ import { NavigationEnd } from '@angular/router';
...
+
+## Use Outside of an Injection Context
+
+The `injectNavigationEnd` function accepts an optional `Injector` parameter, enabling usage outside of an injection context.
+
+```ts
+@Component()
+export class ExampleComponent {
+	private readonly injector = inject(Injector);
+	source$ = injectNavigationEnd(this.injector);
```
Comment (eneajaho) [inline docs/src/content/docs/utilities/navigation-end.md:46]: Shouldn't this be inside ngOnInit to show it correctly?

**Comment by eneajaho** [comment]
Thank you for this!

File: docs/src/content/docs/utilities/navigation-end.md (hunk 1 above)
Comment (tomer953) [reply docs/src/content/docs/utilities/navigation-end.md]: done

**Comment by tomalaforge** [comment]
That's amazing 🚀

Nice work

**Review by nartc — APPROVED** [review]
LGTM!

**Comment by tomer953** [comment]
Thanks :>
I did not have time to refactor all of them, but hopefully new operators will use the updated template.

@nartc It seems that the CI did not updated the website.

**Pull request merged**